lib/system/auditlog/auditlog.go

```diff
@@ -93,7 +92,7 @@ var Domains = []string{
 }
 
 var syscalls = []string{
-	"open", "creat", "rename", "unlink", "mkdir", "rmdir", "chown", "chmod", "symlink",
+	"open", "creat", "rename", "unlink", "mkdir", "rmdir", "chown", "chmod", "symlink", "read",
```

Comment:
I thought I'd add a missing syscall although this was not strictly necessary in the context of this PR.